### PR TITLE
Updated release notes for ACPAnalytics iOS v2.2.4 release

### PR DESCRIPTION
**Pull request by addb**

### CODE REVIEW

File: release-notes.md [hunk 1]
```diff
@@ -8,6 +8,13 @@ description: Release notes and change logs for the Adobe Experience Platform Mob
 
 Please take a moment to fill out a [short survey](https://www.surveymonkey.com/r/AEPDocs) on how we can better assist you with enabling Adobe Experience Cloud solutions and services on your mobile apps.
 
+## June 1, 2020
+
+### iOS Analytics 2.2.4
+
+* Fixed TimeZone calculation bug
```
Comment (nporter-adbe) [inline release-notes.md:15]: I think `TimeZone` should just be timezone

File: using-mobile-extensions/adobe-analytics/adobe-analytics-release-notes.md [hunk 2]
```diff
@@ -1,5 +1,16 @@
 # Release Notes
 
+## June 1, 2020
+
+### @adobe/react-native-acpanalytics 1.1.6
```
Comment (nporter-adbe) [inline using-mobile-extensions/adobe-analytics/adobe-analytics-release-notes.md:5]: Are we including react versions here now?

Comment (addb) [reply using-mobile-extensions/adobe-analytics/adobe-analytics-release-notes.md:5]: We started in Media so thought will do it here as well. Whatsay?

Comment (kc2cheng) [reply using-mobile-extensions/adobe-analytics/adobe-analytics-release-notes.md:5]: I thought we removed the react version even for Media, didn't we?

Comment (addb) [reply using-mobile-extensions/adobe-analytics/adobe-analytics-release-notes.md:5]: We just removed the react from the main release notes but kept in the media release notes.

**Pull request merged**